app/(gcforms)/[locale]/(app administration)/admin/(with nav)/layout.tsx

```diff
@@ -59,7 +52,7 @@ export default async function Layout({
                 </li>
 
                 <li className="mr-5 text-base">
-                  <YourAccountDropdown isAuthenticated={Boolean(session)} />
+                  <YourAccountDropdown isAuthenticated={true} />
```

Comment:
Do we need the `isAuthenticated` prop now?

Reply:
I was going to pull it out but the component is reused in the global Header which is used extensively in the form-builder.